xbmc/settings/windows/GUIControlSettings.cpp

```diff
@@ -314,7 +313,45 @@ CGUIControlSpinExSetting::CGUIControlSpinExSetting(CGUISpinControlEx *pSpin, int
 
   m_pSpin->SetID(id);
 
-  FillControl();
+  const std::string &controlFormat = m_pSetting->GetControl()->GetFormat();
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types

Reply:
Fixed